packages/block-library/src/post-author-biography/style.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-post-author-biography {
+	// This block has customizable padding, border-box makes that more predictable.
+	box-sizing: border-box;
+}
```

Comment:
@aaronrobertshaw 

Can we use css like this for globally?
```
[class*="wp-block-"][style*="border"] {
   box-sizing: border-box;
}
```

Reply:
That looks promising. The catch is that we'd have to make sure there are `wp-block-*` classnames on all elements that receive the inline styles.

Some blocks skip serialization and apply styles on inner elements. 

I pulled up the obvious example of `wp-block-search`, whose inline border styles are applied conditionally on the input and button. Fortunately, these elements have `wp-block-*` classes, so it'd work.

Reply:
Agreed 👍 

We need to review all blocks and verify that box-sizing: border-box is correctly applied (if we use `box-sizing:border-box` globally) 
especially since some blocks have borders on inner elements. 

Can I create separate PR for this?

Reply:
I didn't mean to derail the PR, that's why I suggested it as a good follow-up given the obvious edge cases.

Reply:
Understood. 
I appreciate your suggestion. I agree that it’s important to review and ensure `box-sizing: border-box` is correctly applied across all blocks, especially given the edge cases with borders on inner elements. I will proceed with creating a separate PR to address this follow-up. Thank you for highlighting it!